blockly-ws-search/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": "Blockly Team",
+  "name": "@blockly/plugin-workspace-search",
+  "version": "1.20200304.0",
+  "description": "Implementation of workspace search for Blockly.",
```

Comment:
This description will show up on npm. Perhaps something like:
"A Blockly plugin that adds workspace search support."